src/navigation/MainTabNavigator.js

```diff
@@ -132,7 +132,12 @@ const MainTabNavigator = createBottomTabNavigator(
     AboutStack
   },
   {
-    tabBarOptions: { activeTintColor: colors.accent, inactiveTintColor: colors.primary }
+    tabBarOptions: {
+      activeTintColor: colors.accent,
+      inactiveTintColor: colors.primary,
+      tabStyle: { marginTop: normalize(5) },
+      labelStyle: { margin: normalize(3) }
```

Comment:
I should use OrientationContext here, when orientation === 'landscape' increase spacing between icon and label . 
Like in this example : 
![IMG_0266](https://user-images.githubusercontent.com/43641321/94568464-caf96e80-026c-11eb-80af-cede0673496c.PNG)